src/server/req_rerun.c

```diff
@@ -361,8 +361,8 @@ timeout_rerun_request(struct work_task *pwt)
 	if (pjob->ji_rerun_preq->rq_conn != PBS_LOCAL_CONNECTION) {
 		conn_idx = connection_find_actual_index(pjob->ji_rerun_preq->rq_conn);
 	}
-	reply_text(pjob->ji_rerun_preq, PBSE_INTERNAL,
-		"rerunning job has timed out");
+	reply_text(pjob->ji_rerun_preq, PBSE_NONE,
```

Comment:
Please make the function header doxygen compliant and if possible change the function name because it isn't a timeout anymore.